.github/workflows/intel-cmake.yml

```diff
@@ -122,5 +122,5 @@ jobs:
           CC: ${{ steps.setup-fortran.outputs.cc }}
           CXX: ${{ steps.setup-fortran.outputs.cxx }}
         run: |
-          ctest . --parallel 2 -C ${{ inputs.build_mode }} -V
+          ctest . --parallel 2 -C ${{ inputs.build_mode }} -V -E HDP-dumpv
```

Comment:
Is "dumpv" intentional?  Not "dumpvd" or "dumpvg"?

Reply:
yes it is a regex for dumpvg and dumpvd